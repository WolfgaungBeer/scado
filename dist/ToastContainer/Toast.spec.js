'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _reactTestRenderer = require('react-test-renderer');

var _reactTestRenderer2 = _interopRequireDefault(_reactTestRenderer);

var _Wrapper = require('../../__mocks__/Wrapper');

var _Wrapper2 = _interopRequireDefault(_Wrapper);

var _Toast = require('./Toast');

var _Toast2 = _interopRequireDefault(_Toast);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

var component = _react2.default.createElement(
    'button',
    null,
    'Test'
);
var onClose = function onClose() {};

it('renders the Toast Component correctly', function () {
    var tree1 = _reactTestRenderer2.default.create(_react2.default.createElement(
        _Wrapper2.default,
        null,
        _react2.default.createElement(_Toast2.default, { id: '1', type: 'success', content: 'Text', onClose: onClose })
    )).toJSON();
    expect(tree1).toMatchSnapshot();

    var tree2 = _reactTestRenderer2.default.create(_react2.default.createElement(
        _Wrapper2.default,
        null,
        _react2.default.createElement(_Toast2.default, { id: '1', type: 'error', content: component, onClose: onClose })
    )).toJSON();
    expect(tree2).toMatchSnapshot();
});