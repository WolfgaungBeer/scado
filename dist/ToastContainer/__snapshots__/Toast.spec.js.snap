// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Toast Component correctly 1`] = `
<div
  className="sc-kEYyzF jQpFH sc-eNQAEJ YwarW"
>
  <div
    className="sc-kkGfuU jzSOBF sc-eNQAEJ hbFCXX"
    color="success"
  >
    <i
      className="material-icons  sc-cvbbAY fToWhC"
      color="white"
      onClick={undefined}
      scale="l"
    >
      check_circle
    </i>
  </div>
  <div
    className="sc-iAyFgw fmWRJw sc-eNQAEJ ckEjxQ"
  >
    Text
  </div>
  <div
    className="sc-hSdWYo cffHXk sc-eNQAEJ jqCkHy"
  >
    <div
      className="sc-eHgmQL ksMDBF"
    >
      <i
        className="material-icons  sc-cvbbAY dxhemg"
        color={undefined}
        onClick={[Function]}
        scale={undefined}
      >
        close
      </i>
    </div>
  </div>
</div>
`;

exports[`renders the Toast Component correctly 2`] = `
<div
  className="sc-kEYyzF jQpFH sc-eNQAEJ YwarW"
>
  <div
    className="sc-kkGfuU jZLxSH sc-eNQAEJ hbFCXX"
    color="error"
  >
    <i
      className="material-icons  sc-cvbbAY fToWhC"
      color="white"
      onClick={undefined}
      scale="l"
    >
      error_outline
    </i>
  </div>
  <div
    className="sc-iAyFgw fmWRJw sc-eNQAEJ ckEjxQ"
  >
    <button>
      Test
    </button>
  </div>
  <div
    className="sc-hSdWYo cffHXk sc-eNQAEJ jqCkHy"
  >
    <div
      className="sc-eHgmQL ksMDBF"
    >
      <i
        className="material-icons  sc-cvbbAY dxhemg"
        color={undefined}
        onClick={[Function]}
        scale={undefined}
      >
        close
      </i>
    </div>
  </div>
</div>
`;
